ras_party/models/models.py

```diff
@@ -66,7 +66,9 @@ def from_party_dict(party):
         ba = BusinessAttributes(business_id=b.party_uuid, sample_summary_id=party['sampleSummaryId'])
         ba.attributes = party.get('attributes')
         name = '{runame1} {runame2} {runame3}'.format(**ba.attributes)
+        trading_as = '{tradstyle1} {tradstyle2} {tradstyle3}'.format(**ba.attributes)
         ba.attributes['name'] = ' '.join(name.split())
+        ba.attributes['trading_as'] = ' '.join(trading_as.split())
```

Comment:
Am I missing something here?  ' '.join(trading_as.split()) is used everywhere trading_as is being assigned but surely it just returns the same thing :/